core/tools.js

```diff
@@ -2086,6 +2079,97 @@
 		 * @member CKEDITOR.tools
 		 */
 		object: {
+
+			/**
+			 * List of ECMA3 object properties with obsolete
+			 * [DontEnum](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Properties)
+			 * attribute.
+			 *
+			 * @private
+			 * @since 4.12.0
+			 */
+			DONT_ENUMS: [
```

Comment:
This should be also a `@member` of  `CKEDITOR.tools.object`, without it, it will be in the `tools` section in the docs

![image](https://user-images.githubusercontent.com/1061942/59516284-52c53900-8ec1-11e9-899b-429b33797d81.png)